_posts/2023-06-13-12.0.1-release.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Apache Arrow 12.0.1 Release"
+date: "2023-06-13 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 12.0.1 release.
+This is mostly a bugfix release that includes [**38 resolved issues**][1]
+from [**12 distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## C++ notes
+
+
+## Go notes
+
+
+## Python notes
```

Comment:
@jorisvandenbossche Can you help fill those. The list of issues from the changelog:
```
    GH-35321 - [Python][CI] Skip extension type test failing with pandas 2.0.1 (#35324)
    GH-35821 - [Python][CI] Skip extension type test failing with pandas 2.0.2 (#35822)
    GH-35845 - [CI][Python] Fix usage of assert_frame_equal in test_hdfs.py (#35842)
    GH-35389 - [Python] Fix coalesce_keys=False option in join operation (#35505)
    GH-34789 - [CI][Python] Update the Python versions in our nightly CI matrix (#35548)
    GH-35329 - [Python] Address pandas.types.is_sparse deprecation (#35366)
```